x/inflation/keeper/hooks.go

```diff
@@ -80,6 +83,37 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumb
 		k.SetEpochMintProvision(ctx, newProvision)
 	}
 
+	defer func() {
+		if mintedCoin.Amount.IsInt64() {
+			telemetry.IncrCounterWithLabels(
+				[]string{types.ModuleName, "hook", "allocate", "total"},
+				float32(mintedCoin.Amount.Int64()),
+				[]metrics.Label{telemetry.NewLabel("denom", mintedCoin.Denom)},
+			)
+		}
+		if staking.AmountOf(mintedCoin.Denom).IsInt64() {
```

Comment:
@fedekunze Is this the correct comparison? Cause it seems to be not covered by the tests

Reply:
LGTM